checker/jdk/index/src/java/io/File.java

```diff
@@ -195,7 +195,7 @@ int getPrefixLength() {
      * string for convenience.  This string contains a single character, namely
      * <code>{@link #separatorChar}</code>.
      */
-    public static final @MinLen(1) String separator = "" + separatorChar;
+    public static final /*!MinLen(1)*/ String separator = "" + separatorChar;
```

Comment:
The problem is that separator has a value type of `@StringVal("/")`, so it can't also have `@MinLen(1)`.